cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHandler.java

```diff
@@ -176,6 +178,7 @@ public void create(HttpRequest request, HttpResponder responder,
                      @PathParam("stream") String stream) throws Exception {
 
     String accountID = getAuthenticatedAccountId(request);
+    Id.Stream streamId = Id.Stream.from(accountID, stream);
```

Comment:
I will get the namespaceId from the request once v3 handlers are implemented (separate PR).